src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,161 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final String BOUNDS_EXCEPTION = "This index is not correct: ";
+    private Node head;
+    private Node tail;
```

Comment:
The class Node should be parametrized, so these fields should reflect the parametrization (Node<T>)